src/ar_http_iface_tests.erl

```diff
@@ -736,6 +736,28 @@ get_txs_by_send_recv_test_() ->
 			))
 	end}.
 
+get_tx_status_test() ->
+	ar_storage:clear(),
+	SearchServer = app_search:start(),
+	Peers = ar_network:start(10, 10),
+	ar_node:add_peers(hd(Peers), SearchServer),
+	TX = (ar_tx:new())#tx {tags = [{<<"TestName">>, <<"TestVal">>}]},
+	ar_node:add_tx(hd(Peers), TX),
+	receive after 250 -> ok end,
+	ar_node:mine(hd(Peers)),
+	receive after 1000 -> ok end,
+	{ok, {_, _, Body, _, _}} =
+		ar_httpc:request(
+			<<"GET">>,
+			{127, 0, 0, 1, 1984},
+			"/tx/" ++ binary_to_list(ar_util:encode(TX#tx.id)) ++ "/status"
+		),
+	{Res} = ar_serialize:dejsonify(Body),
+	HashList = ar_node:get_hash_list(hd(Peers)),
+	?assertEqual(true, lists:member({<<"block_height">>, length(HashList) - 1}, Res)),
```

Comment:
Matching bools like this won't give much usable output when the test fails.

How about do this instead:
`?assertEqual({<<"block_height">>, length(HashList) - 1}, lists:keyfind(<<"block_height">>, 1, Res)),`

Or:
```
ResMap = maps:from_list(Res),
?assertMatch(#{<<"block_height">> => length(HashList) - 1}, Res),
```

Reply:
It actually outputs the evaluated expression, e.g.

```
{expression,"lists : member ( { << \"block_height\" >> , length ( HashList ) - 1 } , Res )"},
                                  {expected,false},
                                  {value,true}]}
```

Reply:
Sure, but you can see the expression by opening the source code anyway. What you cannot see in the source code is the value of `Res` and I think you will get it if doing the `?assertMatch` approach. What do you think?

I think we have too many tests that are hard to debug when they fail because they don't expose the incorrect values.

Reply:
I agree, it makes sense to try out `assertMatch`.

Reply:
`assertMatch` does not really allow to ignore part of the map.. But I replaced these assertions with the whole map assertions.